openmrs/apps/reports/DISPENSATION/sql/prescribed_vs_dispensed.sql

```diff
@@ -1,105 +1,105 @@
-    'Not Available 4' AS 'Title 4',
-    0 AS 'prescribed 4',
-    0 AS 'dispensed 4',
-    'Not Available 5' AS 'Title 5',
-    0 AS 'prescribed 5',
-    0 AS 'dispensed 5',
-    'Not Available 6' AS 'Title 6',
-    0 AS 'prescribed 6',
-    0 AS 'dispensed 6',
-    'Not Available 7' AS 'Title 7',
-    0 AS 'prescribed 7',
-    0 AS 'dispensed 7',
-    'Not Available 8' AS 'Title 8',
-    0 AS 'prescribed 8',
-    0 AS 'dispensed 8',
-    'Not Available 9' AS 'Title 9',
-    0 AS 'prescribed 9',
-    0 AS 'dispensed 9',
-    'Not Available 10' AS 'Title 10',
-    0 AS 'prescribed 10',
-    0 AS 'dispensed 10',
-    'Not Available 11' AS 'Title 11',
-    0 AS 'prescribed 11',
-    0 AS 'dispensed 11',
-    'Not Available 12' AS 'Title 12',
-    0 AS 'prescribed 12',
-    0 AS 'dispensed 12',
-    'Not Available 13' AS 'Title 13',
-    0 AS 'prescribed 13',
-    0 AS 'dispensed 13',
-    'Not Available 14' AS 'Title 14',
-    0 AS 'prescribed 14',
-    0 AS 'dispensed 14',
-    'Not Available 15' AS 'Title 15',
-    0 AS 'prescribed 15',
-    0 AS 'dispensed 15',
-    'Not Available 16' AS 'Title 16',
-    0 AS 'prescribed 16',
-    0 AS 'dispensed 16',
-    'Not Available 17' AS 'Title 17',
-    0 AS 'prescribed 17',
-    0 AS 'dispensed 17',
-    'Not Available 18' AS 'Title 18',
-    0 AS 'prescribed 18',
-    0 AS 'dispensed 18',
-    'Not Available 19' AS 'Title 19',
-    0 AS 'prescribed 19',
-    0 AS 'dispensed 19',
-    'Not Available 20' AS 'Title 20',
-    0 AS 'prescribed 20',
-    0 AS 'dispensed 20',
-    'Not Available 21' AS 'Title 21',
-    0 AS 'prescribed 21',
-    0 AS 'dispensed 21',
-    'Not Available 22' AS 'Title 22',
-    0 AS 'prescribed 22',
-    0 AS 'dispensed 22',
-    'Not Available 23' AS 'Title 23',
-    0 AS 'prescribed 23',
-    0 AS 'dispensed 23',
-    'Not Available 24' AS 'Title 24',
-    0 AS 'prescribed 24',
-    0 AS 'dispensed 24',
-    'Not Available 25' AS 'Title 25',
-    0 AS 'prescribed 25',
-    0 AS 'dispensed 25',
-    'Not Available 26' AS 'Title 26',
-    0 AS 'prescribed 26',
-    0 AS 'dispensed 26',
-    'Not Available 27' AS 'Title 27',
-    0 AS 'prescribed 27',
-    0 AS 'dispensed 27',
-    'Not Available 28' AS 'Title 28',
-    0 AS 'prescribed 28',
-    0 AS 'dispensed 28',
-    'Not Available 29' AS 'Title 29',
-    0 AS 'prescribed 29',
-    0 AS 'dispensed 29',
-    'Not Available 30' AS 'Title 30',
-    0 AS 'prescribed 30',
-    0 AS 'dispensed 30',
-    'Not Available 31' AS 'Title 31',
-    0 AS 'prescribed 31',
-    0 AS 'dispensed 31',
-    'Not Available 32' AS 'Title 32',
-    0 AS 'prescribed 32',
-    0 AS 'dispensed 32',
-    'Not Available 33' AS 'Title 33',
-    0 AS 'prescribed 33',
-    0 AS 'dispensed 33',
-    'Not Available 34' AS 'Title 34',
-    0 AS 'prescribed 34',
-    0 AS 'dispensed 34',
-    'Not Available 35' AS 'Title 35',
-    0 AS 'prescribed 35',
-    0 AS 'dispensed 35';
\ No newline at end of file
+SELECT 'ABC/3TC' AS 'Title 1',
```

Comment:
@tmvumbi2 these drugs don't match the existing drugs in our instance, specifically:

- some don't exist
- existing drug AZT is not present
- existing drug AZT/3TC/NVP shows incorrectly as AZT/3TC+NVP

Reply:
Richard will update the template to reflect the correct drug names.

Reply:
New template:

https://docs.google.com/document/d/1-FjSK9WNp40zYvWIA-0LxiZqBIAK7ZiCn4tKI95ew_c/edit